sycl/unittests/assert/assert.cpp

```diff
@@ -514,24 +527,7 @@ void ParentProcess(int ChildPID, int ChildStdErrFD) {
 #endif // _WIN32
 
 TEST(Assert, TestPositive) {
-  // Preliminary checks
-  {
-    sycl::platform Plt;
-    if (Plt.is_host()) {
-      printf("Test is not supported on host, skipping\n");
-      return;
-    }
-
-    if (Plt.get_backend() == sycl::backend::ext_oneapi_cuda) {
-      printf("Test is not supported on CUDA platform, skipping\n");
-      return;
-    }
-
-    if (Plt.get_backend() == sycl::backend::ext_oneapi_hip) {
-      printf("Test is not supported on HIP platform, skipping\n");
-      return;
-    }
-  }
+  sycl::unittest::PiMock::EnsureMockPluginInitialized();
```

Comment:
I have added a comment. In short it is there to ensure that the spawned threads are able to get platforms from the mock plugin. Since they don't need any redefinitions we do not need to create a `PiMock` instance here, so this is used instead.